src/examples/src/widgets/header/Sticky.tsx

```diff
@@ -0,0 +1,64 @@
+import Header from '@dojo/widgets/header';
+import { Link } from '@dojo/framework/routing/Link';
+import { create, tsx } from '@dojo/framework/core/vdom';
+
+const factory = create();
+
+export default factory(function Basic() {
+	return (
+		<virtual>
+			<Header sticky>
+				{{
+					title: () => 'My App',
+					actions: () => [
+						<Link to="#foo">Foo</Link>,
+						<Link to="#bar">Bar</Link>,
+						<Link to="#baz">Baz</Link>
+					]
+				}}
+			</Header>
+			<p style="padding-top: 80px">
```

Comment:
so as to avoid the user needing to add this padding themselves, could we add a domnode to the header that takes up the appropriate space instead?

Reply:
something like https://jsfiddle.net/h8wnfbr7/

Reply:
Smart, on it.